apps/meteor/app/api/server/v1/misc.ts

```diff
@@ -176,15 +176,19 @@ API.v1.addRoute(
 	{ authRequired: true },
 	{
 		async get() {
-			const fields = getDefaultUserFields();
-			const { services, ...user } = (await Users.findOneById(this.userId, { projection: fields })) as IUser;
+			const userFields = { ...getBaseUserFields, services: 1 };
```

Comment:
Isn't `getBaseUserFields` a function?

Reply:
Yep, fixed on local, have to push. 

Working on some other things.

Bad mistake 🙈